Shared/Flows/Onboarding/TextInputViewController.swift

```diff
@@ -73,12 +73,19 @@ class TextInputViewController<ResultType>: ViewController, Sizeable, Completable
                                            for: UIControl.Event.editingChanged)
         self.textEntry.textField.delegate = self
 
-        KeyboardManager.shared.addKeyboardObservers(with: nil)
-        KeyboardManager.shared.$willKeyboardShow.mainSink { [unowned self] willShow in
-            UIView.animate(withDuration: 0.01) {
-                self.view.setNeedsLayout()
```

Comment:
Instead of having different publishers for different events, you can just listen to one current event publisher.